test/test_ssh.yml

```diff
@@ -0,0 +1,12 @@
+
+- name: Ensure that the rule runs with default parameters
+  hosts: all
+  vars:
+    ssh_dump_user: root
+    ssh_dump_server: 192.168.124.1
```

Comment:
where does this IP address come from?

Reply:
This ip address need be provided by tester or customer. If ssh dumping specified, there will be two remote nodes. One is the system where people configure kdump, the other is the ssh server. As soon as the kdump node system crashed, it will boot into a small kdump kernel on the kdump node and try to dump the crashed system's memory to the remote ssh server. Later, people can check the saved memory to analyze what was happening in there.

Reply:
And the test infrastructure provides a ssh server at this address? Otherwise, how do you ensure that the test will succeed?

Reply:
As I said in description of pull request, this test/test_ssh.yml may not pass the CI test. I just made it as an example showing how to use ssh. I have no idea where to put it.